source/win-main/Editor.vue

```diff
@@ -591,22 +591,54 @@ export default {
      */
     editorMousedown (event) {
       if (event.target !== this.$refs.editor) {
-        // return // Only handle if the event's target is the editor itself
+        return 
+      }
+      // if the button is the mouse left button, start selecting lines
+      if(event.button === 0){
+
+        // set the start point of the selection to be where the mouse was clicked
+        let startPoint = this.editor.codeMirror.coordsChar({ left: event.pageX, top: event.pageY});
```

Comment:
You can convert these to `const` (improves readability as it is clear those variables won't be changed)

Reply:
Ah, and: That startpoint should rather be called anchor (because that makes it clear that this is the anchor of the whole selection even as it is being resized) AND also saved as a variable of the Vue component (that is, set this in `data` above).

The reason is that you can then factor out the custom mousemove-event you are listening to after mousedown has been pressed. Inside the mousedown-event (which will be triggered every time) you can then make a check that the anchor is actually set. So here you should set it to the given coordinates as you do, and on mouseup you should set it to `undefined` or `null`. In the mousemove-handler (which should be implemented just like the mousedown and mouseup) you can then at the beginning just check for `anchor !== undefined`